### PR TITLE
Add r2dec

### PR DESCRIPTION
**Pull request by n01e0**
r2dec support

### CODE REVIEW

**Comment by n01e0** [comment]
Decompiled output contains unnecessary comments(`/* r2dec pseudo code output */`), changed to draft.

File: backend/Dockerfile [hunk 1]
```diff
@@ -166,3 +166,27 @@ RUN pip install /mdec-base /mdec-snowman \
  && rm -rf /mdec-base /mdec-snowman
 
 CMD ["python3", "-m", "mdecsnowman"]
+
+#
+# r2dec
+#
+FROM base as r2dec
```
Comment (mborgerson) [inline backend/Dockerfile:173]: Can you please move this section so it is sorted by name, same for other cases below

**Comment by n01e0** [comment]
All files sorted by name.

**Comment by n01e0** [comment]
I would also like to remove the duplicate `#include`.

**Comment by n01e0** [comment]
> I would also like to remove the duplicate `#include`.

Resolved. and added test for r2dec.

**Comment by mborgerson** [comment]
Awesome, thanks

**Pull request merged**